learners/setup.md

```diff
@@ -2,53 +2,136 @@
+conda create -y -n napari-env -c conda-forge python=3.9 bioformats_jar
+
+# Activate the napari-env virtual environment
+conda activate napari-env
+
+# Install napari and plugins using pip
+pip install "napari[all]" napari-aicsimageio
+```
+
+## Post Setup Checks
+
+Before starting the course it is important to check that your setup is working.
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+## Check napari opens a tiff image.
+
+Any napari installation should be able to open a tiff image. Start napari.
+Note that if you're already in the napari-env virtual environment from the
+installation steps it is not necessary to run `conda activate napari-env`.
+```bash
+conda activate napari-env
+napari
+```
+Open a file with
+
+`File > Open file(s)...`
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Navigate to the directory where you saved `00001_01.ome.tiff` earlier on.
+- Select `00001_01.ome.tiff` and click open.
+- If you see a `Choose reader` dialog, select `napari builtins`.
 
-### Details
+::::::::::::::::::::::::::
+::::::::::::::::::: solution
 
-Setup for different systems can be presented in dropdown menus via a `solution`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+## Expected Output
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
+![If this is what you see your napari installation is working as expected.
+ If not then please check the installation or get in touch with the course
+ organisers for help.](fig/ome_00001.png){alt="A screenshot of freshly
+ opened 00001_01.ome.tiff" width='80%'}
 
-:::::::::::::::: solution
+Close napari: `File > exit`
 
-### Windows
+::::::::::::::::::::::::::
 
-Use PuTTY
+:::::::::::::::::::: challenge
 
-:::::::::::::::::::::::::
+## Check napari opens a czi image.
 
-:::::::::::::::: solution
+During the course we'll be working with czi images. To open these the
+napari-aicsimageio plugin is needed. Let's check the plugin is working.
 
-### MacOS
+Start napari. Note that if you've already done the previous test (tiff image)
+then it should not be necessary to run `conda activate napari-env`.
+However, it is necessary to restart napari to ensure the image display
+is correctly formatted.
 
-Use Terminal.app
+```bash
+conda activate napari-env
+napari
+```
 
-:::::::::::::::::::::::::
+`File > Open file(s)...`
 
+- Navigate to the directory where you saved
+`Plate1-Blue-A-12-Scene-3-P3-F2-03.czi` to earlier on.
+- Select `Plate1-Blue-A-12-Scene-3-P3-F2-03.czi` and click open.
 
-:::::::::::::::: solution
+::::::::::::::::::::::::::
+::::::::::::::::::: solution
 
-### Linux
+## Expected Output
 
-Use Terminal
+![If this is what you see your napari and aicsimageio plugin installation
+ is working as expected. If not then please check the installation or get
```

Comment:
What do we mean here by "check the installation"? Can we give more details? Should they double check that each step worked?

Reply:
Added notes on how to check the installation. 4656fec